guides/welfare/ethical_boundaries.md

```diff
@@ -2,4 +2,4 @@
 
 We recognise that there may be certain types of customer, or certain types of industry that people feel uncomfortable working with or in.
 
-At Made Tech nobody will be forced to work on a customer engagement in which they're not comfortable. If you feel this way about any recent customer wins or new engagements we're expecting to kick-off, please let it be known to a Director so we can factor this in to any team shape planning.
+At Made Tech nobody will be forced to work on a customer engagement in which they're not comfortable. If you feel this way about any recent customer wins or new engagements we're expecting to kick-off, please let it be known the Scheduling team so we can factor this in to any team shape planning.
```

Comment:
Missing a 'to' before 'the Scheduling team'

Reply:
Fixed.